### DISCUSSION TITLE
Trace message for autoReplaceStringTemplate logic does not output new content correctly

### DISCUSSION THREAD

**Discussion by pascal-hofmann** [renovatebot/renovate#26130] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovate/renovate:37.74.3

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

Hi,
I'm trying to debug an issue with autoReplaceStringTemplate and enabled log level trace for this. Unfortunately the trace message that should print the content after the replacement only prints `[content]` instead.

How can I get this to print the actual content instead?

(I don't think a reproduction repo makes sense here.)

### Relevant debug logs

<details><summary>Logs</summary>

```

TRACE: packageFile content after autoreplace (repository=…, baseBranch=renovate-improvements, packageFile=config/Dockerfile, branch=renovate/somedep-10.x)
       "content": "[content]"
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Seems like you or someone else would need to contribute a PR to fix the problem before it can print correctly

**Reply by pascal-hofmann** [reply]
I found out that there is a serializer that leads to this behavior:
https://github.com/renovatebot/renovate/blob/c426975b2b6ff4184d58be7d0c95f34c4a72c79e/lib/logger/config-serializer.ts#L22.
